### PR TITLE
Allow to set start/endDate for search

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/androidTest/java/com/owncloud/android/lib/resources/files/SearchRemoteOperationIT.java [hunk 1]
```diff
@@ -306,6 +306,37 @@ public void testPhotoSearchTimestamps() throws IOException {
...
+    @Test
+    public void testPhotoSearchLimitDates() throws IOException {
+        for (int i = 0; i < 10; i++) {
+            String filePath = createFile("image" + i);
+            String remotePath = "/image" + i + ".jpg";
+            assertTrue(new UploadFileRemoteOperation(
+                    filePath,
+                    remotePath,
+                    "image/jpg",
+                    String.valueOf(1464818400 + i))
```
Comment (AlvaroBrey) [inline src/androidTest/java/com/owncloud/android/lib/resources/files/SearchRemoteOperationIT.java:318]: What does this number mean? Maybe best to make it a constant so it has a name at least

**Review by AlvaroBrey — DISMISSED** [review]

File: src/androidTest/java/com/owncloud/android/lib/resources/files/SearchRemoteOperationIT.java (hunk 1 above)
Comment (tobiasKaminsky) [reply src/androidTest/java/com/owncloud/android/lib/resources/files/SearchRemoteOperationIT.java:318]: It is a random unix time. I will adjust it :)

File: src/androidTest/java/com/owncloud/android/lib/resources/files/SearchRemoteOperationIT.java [hunk 2]
```diff
@@ -306,6 +306,39 @@ public void testPhotoSearchTimestamps() throws IOException {
...
+                false,
+                capability);
+
+        // get all
+        RemoteOperationResult<List<RemoteFile>> result = sut.execute(client);
+        assertEquals(10, result.getResultData().size());
+
+        // limit to greater than start / less than end date
+        sut.setStartDate(1464818400 + 2L);
+        sut.setEndDate(1464818400 + 6L);
```
Comment (AlvaroBrey) [inline src/androidTest/java/com/owncloud/android/lib/resources/files/SearchRemoteOperationIT.java:335]: Use the variable here too please!

**Review by AlvaroBrey — DISMISSED** [review]

**Review by AlvaroBrey — APPROVED** [review]

**Pull request merged**